### DISCUSSION TITLE
What's the correct way to setup Homebrew in fish?

### DISCUSSION THREAD

**Discussion by originbeh** [fish-shell/fish-shell#9144] [Q&A]
Apparently the `fish_add_path /opt/homebrew/bin` suggested [here](https://stackoverflow.com/a/70203456) [is not correct](https://github.com/Homebrew/brew/issues/8965) but what's the alternative to:

```
# .zshrc
eval "$(/opt/homebrew/bin/brew shellenv)"
```

in fish?

Thanks.

**Comment by faho** [comment]
Have you *tried* `/opt/homebrew/bin/brew shellenv | source`?
